Uplift/Views/DropdownView.swift

```diff
@@ -45,7 +45,8 @@ class DropdownView: UIView {
     var currentHeight: CGFloat!
```

Comment:
access modifiers on these too

Reply:
`currentHeight` needs to be accessed by superviews in order to properly resize whichever collection views that might use dropdown cells.